.github/workflows/test.yaml

```diff
@@ -73,12 +73,17 @@ jobs:
 
     - name: Run tests
       shell: bash
+      env:
+        TEST_TOKEN: ${{secrets.KBASE_CI_TOKEN}}
       run: |
         export HOMEDIR=`pwd`
         export PYTHONPATH=$HOMEDIR/test:$HOMEDIR/lib:$PYTHONPATH
         export HANDLE_SERVICE_TEST_CFG=$HOMEDIR/test.cfg
         pytest --cov=lib/AbstractHandle --cov-report=xml test
 
+        # run handle service container test
+        sh container_test/run_tests.sh
+
```

Comment:
Why not make this another job so it runs in parallel, similar to the workspace tests?

Reply:
then I need to reinstall python, requests, and pytest, which are installed already.

Reply:
Worth it to parallelize the test

Reply:
👍

Reply:
:+1: